src/coreclr/debug/daccess/request.cpp

```diff
@@ -3497,7 +3497,7 @@ ClrDataAccess::TraverseVirtCallStubHeap(CLRDATA_ADDRESS pAppDomain, VCSHeapType
     SOSDacEnter();
 
     BaseDomain* pBaseDomain = PTR_BaseDomain(TO_TADDR(pAppDomain));
-    VirtualCallStubManager *pVcsMgr = PTR_VirtualCallStubManager((TADDR)pBaseDomain->GetLoaderAllocator()->GetVirtualCallStubManager());
+    VirtualCallStubManager *pVcsMgr = pBaseDomain->GetLoaderAllocator()->GetVirtualCallStubManager();
```

Comment:
Any reason to get rid of the Target to Host cast? Looks OK in this case since all the fields are PTRs, but seems brittle.

Reply:
Nvm, saw the latter changes.